packages/packages/store/src/index.ts

```diff
@@ -0,0 +1,113 @@
+import {
+	createSlice,
+	ReducersMapObject,
+	configureStore,
+	combineReducers,
+	Middleware,
+	Store,
+	Slice,
+	AnyAction,
+	CreateSliceOptions,
+} from '@reduxjs/toolkit';
+
+export type {
+	Slice,
+	CreateSliceOptions,
+	PayloadAction,
+	Store,
+	Dispatch,
+	AnyAction,
+	Action,
+	MiddlewareAPI,
+	Middleware,
+} from '@reduxjs/toolkit';
+
+export { createSlice } from '@reduxjs/toolkit';
```

Comment:
why do you export createSlice if you're creating the slice yourself?

Reply:
Removed.